android_configure.py

```diff
@@ -0,0 +1,76 @@
+from platform import platform
+import sys
+import os
+
+# TODO: In next version, it will be a JSON file listing all the patches, and then it will iterate through to apply them.
+def patch_android():
+    print("- Patches List -")
+    print("[1] [deps/v8/src/trap-handler/trap-handler.h] related to https://github.com/nodejs/node/issues/36287")
+    if platform().startswith('Linux'):
+        os.system('patch -f ./deps/v8/src/trap-handler/trap-handler.h < ./android-patches/trap-handler.h.patch')
+    print("\033[92mInfo: \033[0m" + "Tried to patch.")
+
+if platform().startswith('Windows'):
+    print('android-configure is not supported on Windows yet.')
+    sys.exit(1)
+
+if len(sys.argv) == 2 and sys.argv[1] == "patch":
+    patch_android()
+    sys.exit(0)
+
+if len(sys.argv) != 4:
+    print("Usage: ./android-configure [patch] <path to the Android NDK> <Android SDK version> <target architecture>")
```

Comment:
Yes, but in this branch, BUILDING.md is out of date, do I need to create a new branch and pull requests to update it or later?

Reply:
I think it would be better to update this doc in the same pr.

Reply:
Hi,  @MeowShe. Would you update BUILDING.md in this pr?

Reply:
I want but I don't seem to know how to do it because the BUILDING.md in this branch is out of date and if I update BUILDING.md it will cause a conflict and I will have to merge main branch to solve it, but the nodejs Pull Request CI doesn't seem to allow me to do that.

Reply:
Do you know how to rebase upstream nodes? If you know, then rebase instead of merge. If you don't know how to rebate,  then just merge, It is ok, I will land your pr manually.

Reply:
Thanks, I will update BUILDING.md using rebase when all the fix are ready.

Reply:
@MeowShe would you mind update the BUILDING.md to keep the doc update with this change?

Reply:
Thanks. A person asked me in an email if android configure is supported in the latest main branch. So I remember our BUILING.md if lost updated with our codebase.😃